app/build.gradle

```diff
@@ -12,11 +12,6 @@ dependencies {
   implementation project(':persistence:model')
 
   implementation project(':domain-rrd:rrd-shared')
```

Comment:
Why are we keeping `rrd-shared`?

Reply:
there's some data objects that are used by several different Gradle subprojects. my plan right now is to extract these out in a follow up PR

Reply:
for more clarity here, many of the shared objects are related to BIP, so I will extract those files into a new gradle subproject that @greene858 is creating